docs/features/software-catalog/configuration.md

```diff
@@ -44,6 +44,16 @@ When multiple `catalog-info.yaml` files with the same `metadata.name` property
 are discovered, one will be processed and all others will be skipped. This
 action is logged for further investigation.
 
+Locations in the root of your backstage repository, e.g. [catalog examples](https://github.com/backstage/backstage/tree/master/packages/catalog-model/examples), 
+assume the location of the backend instance and must walk the tree upwards two times.
```

Comment:
I think this is an opportunity to introduce the concept for `type: file` locations as well. Noting that you're able to use them to bring in content for the file system but that it is best to limit that to local development, test setups and example data. You're also not able to use placeholders like `$text:` within them. Then mention the quirk the path you specify is relative to the running backend process, typically `packages/backend`.